site/_variables.yml

```diff
@@ -1,44 +1,74 @@
 # VARIABLE NAMING CONVENTIONS:
-# - Variables for long & short forms should be related (e.g. `developer_framework`, `developer`)
-# - Variable names should be descriptive and consistent (e.g. `user_first_name`, `user_last_name`)
-# - Use prefixes / suffixes for grouping (e.g. `order_date`, `order_amount`)
-# - Use snake case (e.g. `developer_framework`)
-# - Avoid abbreviations (e.g. `customer_address` NOT `cust_addr`) — `vm` for ValidMind is one exception
-# - Include enough contextual information (e.g. `invoice_total` NOT `total`)
+# - Variables for long & short forms should be related (e.g. `validmind`, `vm`)
+# - Variable names should be descriptive and consistent (e.g. `user.firstname`, `user.lastname`)
+# - Use prefixes / suffixes for grouping (e.g. `order.date`, `order.amount`)
+# - Use period delimiter & nested array structure to differentiate variable types (e.g. `validmind.developer` — Note that Quarto limits this to one nesting level): https://quarto.org/docs/authoring/variables.html
+# - Try to avoid abbreviations (e.g. `customer.address` NOT `cust.addr`) — Exceptions can be made for product/branding or standard conceptual phrases (e.g. `vm` for ValidMind, `aws` for Amazon Web Services)
+# - Include enough contextual information (e.g. `invoice.total` NOT `total`)
+
+
 
 # PRODUCT NAMES
 # Names of ValidMind products or core features
 
 # Long form
-validmind_platform: "ValidMind Platform UI"
-validmind_developer: "ValidMind Developer Framework"
-risk_platform: "ValidMind AI risk platform"
+# Rendered format: `{{< var validmind.platform >}}`
+validmind:
+  platform: "ValidMind Platform UI"
+  developer: "ValidMind Developer Framework"
+  product: "ValidMind AI risk platform"
 
 # Short form
-platform: "platform UI"
-developer: "developer framework"
+# Rendered format: `{{< var vm.platform >}}`
+vm:
+  platform: "platform UI"
+  developer: "developer framework"
+  product: "ValidMind"
 
 
 # PRODUCT URLS
 # Links to ValidMind hosted web applications
-# This section is always prefaced by `vm_`
 
 # Long form — Text & URL
-vm_contact: "[**Contact Us**](https://validmind.com/contact/)"
+# Rendered format: `{{< var link.contact >}}`
+
+link:
+  contact: "[**Contact Us**](https://validmind.com/contact/)"
+  login: "[Log in to ValidMind](/guide/configuration/log-in-to-validmind.qmd)."
 
 # Short form — URL only
-vm_url_us1: "https://app.prod.validmind.ai"
-vm_url_ca1: "https://app.ca1.validmind.ai"
+# Rendered format: `{{< var url.us1 >}}`
+
+url:
+  us1: "https://app.prod.validmind.ai"
+  ca1: "https://app.ca1.validmind.ai"
+
+# Amazon Web Services Privatelink specific URLs
+# Rendered format: `{{< var aws.privatelink >}}`
+
+aws:
+  privatelink: "https://private.prod.vm.validmind.ai"
+
+# Google Cloud Private Service Connect specific URLs
+# Rendered format: `{{< var psc.validmind >}}`
+
+psc:
```

Comment:
This `psc` acronym doesn't quite make sense to me and I don't think anyone will ever remember it. The convention you follow on lines 49-50 feels more elegant as it starts with the cloud provider and then works its way down to the service. 

How about this? 

```yml
gcp:
  private_service_connect: "https://private.us3-prod.validmind.ai"
  private_authentication: "https://auth.private.validmind.ai"
```

(FYI, Spencer has said that the authentication endpoint might go away at some point in the future.)

Reply:
I was torn between `psc` and `gcp` or `vpc`... the last one seems more generic, but then I feel like the aws should go under that as well. What about:

```
gcp:
    connect: ...
    authentication: ...
```

I'm really not a fan of the super long snake_case ones, as that feels even MORE tedious than just typing out the values in some cases! I had a version where I had a "rule" where I discussed snake_case, but the more I thought about it the more it went against us trying to simplify it...

Reply:
> What about:
> 
> ```
> gcp:
>     connect: ...
>     authentication: ...
> ```
> 
> I'm really not a fan of the super long snake_case ones, as that feels even MORE tedious than just typing out the values in some cases! 

+1 to this suggestion — both the cloud provider context and the fact that it's a private connection are clear. Agreed on avoiding long variable names.